### DISCUSSION TITLE
Codewars with colleagues

### DISCUSSION THREAD

**Discussion by rezgar-muhammad** [codewars/codewars.com#3088] [Q&A]
Hello,
I wanted to ask if codewars has the option to create teams or group to compete between colleagues?

Thanks in advance and Have a nice day

**Comment by Sad-Abd** [comment]
Hello @rezgar-muhammad,

Yes, It has. Please refer to [Clan](https://docs.codewars.com/community/following#clans) section in official [Documentation](https://docs.codewars.com) for more information.
